README.md

```diff
@@ -30,31 +31,31 @@ Name your file with your GitHub username `username.json`; for example mine was `
 ```
 Notes on the information: 
 
-**Name**: You can use your github name, actual name, or a pseudonym! 
-
+ > * **Name**: You can use your github name, actual name, or a pseudonym! 
```

Comment:
Could this be changed to -
```
 > * **Name**: You can use your github name, actual name, or a pseudonym! 
 > * **Url**: Please insert Github link here, we will expand the user json files to include socials in the future
 > * **Image**: You can use any *appropriate* profile image here
 > * **Why** : Please write down your reason why you have joined the #FitDevs community!
 > * **Weekly**: Your goals for fitness, can be food/exercise/mental/emotional, whatever fitness means to YOU 
```

 > * **Name**: You can use your github name, actual name, or a pseudonym! 
 > * **Url**: Please insert Github link here, we will expand the user json files to include socials in the future
 > * **Image**: You can use any *appropriate* profile image here
 > * **Why** : Please write down your reason why you have joined the #FitDevs community!
 > * **Weekly**: Your goals for fitness, can be food/exercise/mental/emotional, whatever fitness means to YOU 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix